### PR TITLE
Avatar methods

### PR DESCRIPTION
**Pull request by tsilva**

### CODE REVIEW

File: src/eadapters/base.py [hunk 1]
```diff
@@ -88,6 +88,9 @@ def login_account(self, username, password, *args, **kwargs):
     def me_account(self, *args, **kwargs):
         raise appier.NotImplementedError()
 
+    def avatar_me_account(self, *args, **kwargs):
```
Comment (joamag) [inline src/eadapters/base.py]: avatar is not crud so it comes after crud methods

File: src/eadapters/budy_a.py [hunk 2]
```diff
@@ -151,6 +151,10 @@ def me_account(self, *args, **kwargs):
         account = models.BDAccount.me()
         return account
 
+    def avatar_me_account(self, *args, **kwargs):
```
Comment (joamag) [inline src/eadapters/budy_a.py]: avatar is not crud so it comes after crud methods

File: src/eadapters/models/budy/bd_account.py [hunk 3]
```diff
@@ -56,6 +56,13 @@ def me(cls):
         account = api.me_account()
         return cls.wrap(account)
 
+    @classmethod
+    @bd_common.handle_error
+    def avatar_me(cls):
```
Comment (joamag) [inline src/eadapters/models/budy/bd_account.py]: avatar is not crud so it comes after crud methods

**Comment by tsilva** [comment]
@joamag fixed

**Pull request merged**